src/pages/EditorTeam/EditorTeam.tsx

```diff
@@ -0,0 +1,300 @@
+import { TextField, MenuItem, Modal, Input } from '@material-ui/core';
+import Button from '@material-ui/core/Button';
+import IconButton from '@material-ui/core/IconButton';
+import { makeStyles } from '@material-ui/core/styles';
+import Table from '@material-ui/core/Table';
+import TableBody from '@material-ui/core/TableBody';
+import TableCell from '@material-ui/core/TableCell';
+import TableHead from '@material-ui/core/TableHead';
+import TableRow from '@material-ui/core/TableRow';
+import DeleteIcon from '@material-ui/icons/Delete';
+import DoneIcon from '@material-ui/icons/DoneAllTwoTone';
+import EditIcon from '@material-ui/icons/EditOutlined';
+import RevertIcon from '@material-ui/icons/NotInterestedOutlined';
+
+import React, { useState, useEffect, FC } from 'react';
+
+import { useSelector } from 'react-redux';
+
+import { APIPerson } from 'models/registry.models';
+
+import Navigation from 'shared/Navigation/Navigation';
+import { RootState } from 'store/models/rootstate.model';
+
+import { deleteTeamMember, editTeamMember } from '../../helpers/editing-db.handler';
+
+import NewTeamMember from './components/NewTeamMember';
+import styles from './EditorTeam.module.css';
+
+
+const useStyles = makeStyles(theme => ({
+  root: {
+    width: '100%',
+    marginTop: theme.spacing(3),
+    overflowX: 'auto'
+  },
+  table: {
+    minWidth: 650
+  },
+  selectTableCell: {
+    width: 60
+  },
+  tableCell: {
+    width: 130,
+    height: 40
+  },
+  input: {
+    width: 130,
+    height: 40
+  }
+}));
+
+interface IPerson extends APIPerson{
+  lp?: number;
+  isEditMode?: boolean;
+}
+
+interface IProps {
+  row: IPerson
+  name: string;
+  onChange: any;
+}
+
+
+const CustomTableCell = ({ row, name, onChange }: IProps) => {
```

Comment:
This file is over 300 lines length. Maybe we should split it into smaller files?